driver/config/.schema/config.schema.json

```diff
@@ -1488,6 +1488,12 @@
             "minLength": 16
           },
           "uniqueItems": true
+        },
+        "aes": {
```

Comment:
It would be great if this could support key rotation like we do for cookies and other secrets

Reply:
done